src/Downloader.ts

```diff
@@ -171,13 +169,11 @@ class Downloader {
       if (!this.mwCapabilities.restApiAvailable || !this.mwCapabilities.veApiAvailable) {
         logger.log(`Using local MCS and ${this.mwCapabilities.veApiAvailable ? 'remote' : 'local'} Parsoid`);
         await this.initLocalServices();
-        const domain = (urlParser.parse(this.mw.base)).host;
+        const domain = urlParser.parse(this.mw.base).host;
         this.restApiUrl = `http://localhost:6927/${domain}/v1/page/mobile-sections/`;
 
         if (!this.mwCapabilities.veApiAvailable) {
-          const webUrlHost = urlParser.parse(this.mw.webUrl).host;
-          // todo will be changed in #1154
-          this.parsoidFallbackUrl = `http://localhost:8000/${webUrlHost}/v3/page/pagebundle/`;
+          this.veApiUrl = `http://localhost:8000/${domain}/v3/page/pagebundle/`;
```

Comment:
Are you really sure that the local parsoid provides a veApiUrl? I believe this is a restApiUrl.